src/week1/day0/css/style.css

```diff
@@ -0,0 +1,72 @@
+body{
+   font-size: 14px;
+   font-family: serif;
+}
+
+.container1{
+   display: flex;
+   flex-flow: row;
+}
+
+header{
+   background: black;
+   color: white;
+   text-align: center;
+}
+
+nav{
+   width: 25%;
+   background: orange;
+   color: white;
+}
+
+section{
+   width: 50%;
+   background: lightblue;
+}
+
+article{
+   background: lightgrey;
+}
+
+aside{
+   width: 25%;
+   background: blue;
+   color: white;
+}
+
+footer{
+   text-align: center;
+   background: yellow;
+}
+
+.header{
+   font-size: 46px;
+}
+
+.footer, .sidebar{
+   font-size: 10px;
+}
+
+.navigation{
+   font-size: 12px;
+}
+
+.content *{
```

Comment:
font is an inherited property, there is no need to include the wildcard selector